addons/ui/org.openhab.ui.cometvisu/README.md

```diff
@@ -11,16 +11,26 @@ This adds a backend for the web based visualization CometVisu (http://www.cometv
 
 ## Installation
 
-* Install the CometVisu addon (optionally you can also installe the CometVisu-PHP addon, but it is not required to run the CometVisu)
+* Install the CometVisu addon (optionally you can also install the CometVisu-PHP addon, but it is not required to run the CometVisu)
+* Enable the auto-download feature (available since 2.2.0) via configuration setting *OR*
 * Download the CometVisu release (https://github.com/CometVisu/CometVisu/releases) and extract the "release" folder (the one which contains the index.html file) of the downloaded archive somewhere on your openHAB2 server (make sure that openHAB can access that folder). 
 
-Note: You have to set the path to this folder in your config file (webFolder).
+Note: You have to set the path to this folder in your configuration (webFolder).
 
 ## Configuration
 
 ### Server configuration
 
-You can customize some setting of the CometVisu backend in a configuration file named 'cometvisu.cfg' which has to be created in the 'conf/services/' folder of your openHAB2 instance, if it does not exist.
+#### Version >= 2.2.0
```

Comment:
The docs should always only describe the latest version - after all, that's why they are in a versioning system.